> There is a formatting issue, please run `mvn process-sources` and amend your commit.

I got this problem:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (enforce) on project quarkus-test-extension-deployment: Execution enforce of goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce failed: org.apache.maven.shared.dependency.graph.DependencyGraphBuilderException: Could not resolve following dependencies: [io.quarkus:quarkus-test-extension:jar:999-SNAPSHOT (compile)]: Could not resolve dependencies for project io.quarkus:quarkus-test-extension-deployment:jar:999-SNAPSHOT: Could not find artifact io.quarkus:quarkus-test-extension:jar:999-SNAPSHOT in oss.sonatype.org (https://oss.sonatype.org/content/repositories/snapshots/)

--- a/extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/reactive/ReactiveMongoDatabase.java
+++ b/extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/reactive/ReactiveMongoDatabase.java
@@ -1,6 +1,5 @@
 package io.quarkus.mongodb.reactive;
 
-import com.mongodb.reactivestreams.client.MongoDatabase;
 import java.util.List;
 
 import org.bson.Document;
@@ -11,6 +10,7 @@ import com.mongodb.client.model.CreateCollectionOptions;
 import com.mongodb.client.model.CreateViewOptions;
 import com.mongodb.client.model.changestream.ChangeStreamDocument;
 import com.mongodb.reactivestreams.client.ClientSession;
+import com.mongodb.reactivestreams.client.MongoDatabase;
 
 import io.quarkus.mongodb.AggregateOptions;
 import io.quarkus.mongodb.ChangeStreamOptions;
@@ -565,7 +565,6 @@ public interface ReactiveMongoDatabase {
      */
     <T> Multi<T> aggregate(ClientSession clientSession, List<? extends Bson> pipeline, Class<T> clazz,
             AggregateOptions options);
-
 
     /**
      * @return the underlying database.

